index.ts

```diff
@@ -83,8 +83,8 @@ type Fulfilled<T> = T extends Promise<infer R> ? R : T;
 
   @method hash
   @public
-  @param {Object} object
-  @return {Promise} promise that is fulfilled when all properties of `promises`
```

Comment:
this is due to the potential conflict between type info in JSDoc comments and TS type annotations. my preference is to have a single signal